src/FixedDataTableRow.js

```diff
@@ -158,6 +158,13 @@ class FixedDataTableRowImpl extends React.Component {
      * )
      * ```
      */
+    zIndex: PropTypes.number,
+
+    /**
+     * The vertical position where the row should render itself
+     */
+    offsetTop: PropTypes.number.isRequired,
+
```

Comment:
Earlier we were seperating zIndex and offsetTop and adding there styling for the row in the wrapper div .To remove that wrapper div we have to maintain it's styles in the div of FixedDataTableRowImpl only .So, we will need these variables .